File: 01.JS-Basic/По сложни проверки Упражнение/Journey.js
function journey(input) {

    let budget = Number(input[0])
    let season = input[1]

    let money = 0.0
    let type = ""
    let destination = ""

    if (budget <= 100) {
        destination = "Bulgaria"
        switch (season) {
            case "summer": money = budget * 0.3
                type = "Camp"; break;
            case "winter": money = budget * 0.7
                type = "Hotel"; break;
        }
    }
    else if (budget <= 1000) {
        destination = "Balkans"
        switch (season) {
            case "summer": money = budget * 0.4
                type = "Camp"; break;
            case "winter": money = budget * 0.8
                type = "Hotel"; break;
        }
    }
    else if (budget > 1000) {
        destination = "Europe"
        money = budget * 0.9
        type = "Hotel"
    }

    console.log(`Somewhere in ${destination}`)
    console.log(`${type} - ${money.toFixed(2)}`)
}

journey(["312", "summer"])

File: JAVA-Basic/JavaBasic/src/ConditionalStatementsAdvanced/Exercise/Journey.java
package ConditionalStatementsAdvanced.Exercise;

import java.util.Scanner;

public class Journey {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        double budget = Double.parseDouble(scan.nextLine());
        String season = scan.nextLine();
        String destination = "";
        String destinationType = "";
        double price = 0.0;

        if (budget <= 100) {
            destination = "Bulgaria";
            switch (season) {
                case "summer":
                    destinationType = "Camp";
                    price = budget * 0.30;
                    break;
                case "winter":
                    destinationType = "Hotel";
                    price = budget * 0.70;
                    break;
            }
        } else if (budget <= 1000) {
            destination = "Balkans";
            switch (season) {
                case "summer":
                    destinationType = "Camp";
                    price = budget * 0.40;
                    break;
                case "winter":
                    destinationType = "Hotel";
                    price = budget * 0.80;
                    break;
            }
        } else {
            destination = "Europe";
            destinationType = "Hotel";
            price = budget * 0.90;
        }
        System.out.printf("Somewhere in %s\n", destination);
        System.out.printf("%s - %.2f\n", destinationType, price);
    }
}
